src/app/components/main-list/main-list.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, OnInit } from '@angular/core';
+import { AdsService } from '../../services/ads.service';
+
+@Component({
+  selector: 'app-main-list',
+  templateUrl: './main-list.component.html',
+  styleUrls: ['./main-list.component.scss']
+})
+export class MainListComponent implements OnInit {
+
+  timer: any;
+  adUrl: any;
+  songName: any;
+  ads: any;
+  songImg: any;
+
+  songList=[];
+  adList = [];
+  index1 = 1;
+  index2 = 1;
+  songs = [{
+    name: 'Varona',
```

Comment:
a lot of these properties are no longer being used and are not necessary.  we should clean them up.